WordPressEditor/src/main/java/org/wordpress/android/editor/EditorFragment.java

```diff
@@ -740,13 +738,13 @@ public void run() {
                 if (URLUtil.isNetworkUrl(mediaUrl)) {
                     String mediaId = mediaFile.getMediaId();
                     mWebView.execJavaScriptFromString("ZSSEditor.insertImage('" + mediaUrl + "', '" + mediaId + "');");
-                    AnalyticsTracker.track(Stat.EDITOR_ADDED_PHOTO_VIA_WP_MEDIA_LIBRARY);
+                    mEditorFragmentListener.onTrackableEvent(TrackableEvent.NETWORK_MEDIA_ADDED);
```

Comment:
I mean that `TrackableEvent` should have `NETWORK_IMAGE_ADDED` instead of `NETWORK_MEDIA_ADDED`, and the same for `LOCAL_`. It's already mapping to the correct `Stat` in the gist:

https://gist.github.com/maxme/5b366a79dec750ed1ca7#file-ontrackableevent-java-L13-L18

It should just be more specific in the editor's `TrackableEvent` so we can add in separate video events later, which will map to the already existing `EDITOR_ADDED_VIDEO_VIA_WP_MEDIA_LIBRARY` and `EDITOR_ADDED_VIDEO_VIA_LOCAL_LIBRARY`.